src/utils/helperFunctions/metadataFilters.ts

```diff
@@ -1,25 +1,46 @@
 import { DataStreamWithRoutes } from "src/utils/api/dataStreams";
-import { Route } from "src/utils/api/routes";
+import { RouteWithPermissions } from "src/utils/api/routes";
 
 export const getDataStreamIds = (data: DataStreamWithRoutes[]) => {
```

Comment:
Should you set the return time of this function to a string array?

Reply:
Return types are inferred in this case, not necessary. But I'll add for clarity.